src/code/z_view.c

```diff
@@ -510,7 +541,7 @@ s32 func_800AB944(View* view) {
     return 1;
 }
 
-s32 func_800AB9EC(View* view, s32 arg1, Gfx** gfxp) {
+s32 View_ApplyTo(View* view, s32 mask, Gfx** gfxp) {
```

Comment:
Anything starting with `View_Init` would look to me like it's initializing the view itself, rather than using it to initialize something else.

I see how `Apply` could be confusing, especially as it's a word also used for doing matrix multiplication, so am happy to use a different word.

I do quite like the `To` suffix on the function that takes an argument to operate on, though, with the non-`To` version operating on an implicit target. That feels neat to me.

Reply:
I'm ok with `View_ApplyTo`. It's `View_Apply` but *to* a specific buffer 😃 

Another example of this add-to-implicit/add-to-explicit split can be `SkelAnime_DrawOpa` (implicit draw to opa, no argument for the buffer like `View_Apply`) / `SkelAnime_Draw` (argument for the buffer to append to, like `View_ApplyTo`)

More on that pattern:
`VisMono_Draw` takes a buffer to append to
`Gfx_DrawDListOpa`/`Gfx_DrawDListXlu` append to opa/xlu

Overall to go with these examples this should have `View_ApplyTo` -> `View_Apply` and `View_Apply` -> `View_ApplyOpaXlu`(`OverlayDisp`)
But I don't see `View_Apply` being `View_ApplyOpaXluOverlayDisp` especially since the `OverlayDisp` part depends on the flags

So I'm still ok with Apply and ApplyTo. it's not a big inconsistency deal, it also may only look inconsistent due to the examples I picked